addons/survey/survey.py

```diff
@@ -201,7 +208,9 @@ def _get_result_url(self, cr, uid, ids, name, arg, context=None):
             'Email Template', ondelete='set null'),
         'thank_you_message': fields.html('Thank you message', translate=True,
             help="This message will be displayed when survey is completed"),
-        'quizz_mode': fields.boolean(string='Quiz mode')
+        'quizz_mode': fields.boolean(string='Quiz mode'),
+        'is_answered': fields.function(_is_answered,
+            string="Is Answered", type="boolean"),
```

Comment:
This new field makes no sense, you could simply check that tot_start_survey (or tot_comp_survey depending on your needs) is > 0